azkaban-web-server/src/main/java/azkaban/webapp/servlet/ProjectManagerServlet.java

```diff
@@ -1809,6 +1827,19 @@ private boolean hasPermissionToCreateProject(final User user) {
     return false;
   }
 
+  private boolean hasPermissionToUploadProject(final User user) {
+    for (final String roleName : user.getRoles()) {
```

Comment:
This project follows a different style. Please take a look at the project readme.